docs/dashboards/frequently-asked-questions.md

```diff
@@ -0,0 +1,108 @@
+FAQ
+===
+### How to connect store to the other components
+First, you need to create the correct type of store using the right constructor, and pass the data reference. More about this topic in the DataStore
+<LINK TO THE DATA STORE>
+After that, you need to pass the store to the component config, and that’s it.
+[Here is the demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/dashboard-minimal).
+
+### How to connect component to a cell
+Each cell has a field called id. The same id must be passed in the component config to the `cell` field. Example configuration of component and cell:
+
+```js
+  gui: {
+        enabled: true,
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }]
+            }]
+        }]
+    },
+    components: [
+        {
+            cell: 'dashboard-col-0',
+            type: 'Highcharts',
+            chartOptions: {
+                chart: {
+                    type: 'pie'
+                },
+                series: [{data: [1,2,3]}]
+            },
+        }]
+```
+
+[Here is the demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/component-highcharts).
+
+
+
+
+### Synchronization of components
+To synchronize components you have to specify which event you want to synchronize between each component, as well as they have to use the same store.
```

Comment:
I would name at least the possible synchronization between components.

Reply:
They are named in the section below 🤷

Reply:
Maybe move the section up and write "See next question for possible synchronization events".